cmd/openshift-install/destroy.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/openshift/installer/pkg/destroy"
 	"github.com/openshift/installer/pkg/destroy/bootstrap"
 	_ "github.com/openshift/installer/pkg/destroy/libvirt"
+	_ "github.com/openshift/installer/pkg/destroy/openstack"
```

Comment:
I don't see this package anywhere (and my build fails because of that). Is it possible you've left it out in the squash/rebase?

Reply:
@tomassedovic oops, yes you're right, the new package got lost due to the most recent rebase, I'll re-add it and rebase again now #588 landed, thanks for spotting it!